fpga_interchange/nextpnr_emit.py

```diff
@@ -44,8 +44,14 @@ def main():
     else:
         global_buffers = []
 
+    if 'disabled_routethroughs' in device_config:
+        disabled_routethroughs = device_config['disabled_routethroughs']
+    else:
+        disabled_routethroughs = []
+
```

Comment:
I think this can be re-written as follows:
```suggestion
     disabled_routethroughs = device_config.get('disabled_routethroughs', [])
```

Reply:
thanks, fixed